sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -106,91 +106,39 @@ namespace Azure
-        public static explicit operator bool? (Azure.Core.JsonData json) { throw null; }
-        public static explicit operator double? (Azure.Core.JsonData json) { throw null; }
-        public static explicit operator int? (Azure.Core.JsonData json) { throw null; }
-        public static explicit operator long? (Azure.Core.JsonData json) { throw null; }
-        public static explicit operator float? (Azure.Core.JsonData json) { throw null; }
-        public static explicit operator float (Azure.Core.JsonData json) { throw null; }
-        public static explicit operator string (Azure.Core.JsonData json) { throw null; }
-        public static implicit operator Azure.Core.JsonData (bool value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (double value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (int value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (long value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (bool? value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (double? value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (int? value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (long? value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (float? value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (float value) { throw null; }
-        public static implicit operator Azure.Core.JsonData (string? value) { throw null; }
-        public static bool operator !=(Azure.Core.JsonData? left, string? right) { throw null; }
-        public static bool operator !=(string? left, Azure.Core.JsonData? right) { throw null; }
-        public void Set(string propertyName, bool value) { }
-        public void Set(string propertyName, double value) { }
-        public void Set(string propertyName, int value) { }
-        public void Set(string propertyName, long value) { }
-        public Azure.Core.JsonData Set(string propertyName, object? serializable) { throw null; }
-        public Azure.Core.JsonData Set(string propertyName, object? serializable, System.Text.Json.JsonSerializerOptions options) { throw null; }
-        public void Set(string propertyName, float value) { }
-        public void Set(string propertyName, string? value) { }
-        public Azure.Core.JsonData SetEmptyArray(string propertyName) { throw null; }
-        public Azure.Core.JsonData SetEmptyObject(string propertyName) { throw null; }
-        public Azure.Core.JsonData Set<T>(string propertyName, T[] serializable) { throw null; }
-        public Azure.Core.JsonData Set<T>(string propertyName, T[] serializable, System.Text.Json.JsonSerializerOptions options) { throw null; }
+        public static bool operator ==(Azure.Core.Dynamic.JsonData? left, int right) { throw null; }
```

Comment:
Is this the right list of types to support with equality operators?
- string
- bool
- int
- double
- long
- float

Reply:
Cool, I'll add these.